TreeDB/collections/column_hnsw_search_pack_search.go

```
@@ -346,14 +339,408 @@ func (v *columnHNSWSearchPackPreparedView) searchCosineWithContextTrace(ctx cont
+	visitEpoch := scratch.visitEpoch
+	visitMarks[entryOrdinal] = visitEpoch
+	if trace != nil {
+		trace.ScoreOrdinals = append(trace.ScoreOrdinals, uint32(entryOrdinal))
+	}
+	if err := v.scoreAndPushFrontierVisited(normalizedQuery, entryOrdinal, efSearch, opts.ScoreBatchMode, scratch, &stats, &visitedCandidates, trace); err != nil {
+		return nil, stats, err
+	}
+	nextSeed := 0
+	rowCount64 := uint64(rowCount)
+	traversalSteps := 0
+	termination := ""
```

golangci-lint (2.12.2)

[error] assigned to termination, but reassigned without using the value

(wastedassign)